File: WebApp/Scripts/app/model/CD0005D.js
Ext.define('WEBAPP.model.CD0005D', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'SEQ', type: 'string' }, //項次(畫面)
        { name: 'MMCODE', type: 'string' }, //院內碼(畫面)
        { name: 'MMNAME_C', type: 'string' }, //中文品名(畫面)
        { name: 'MMNAME_E', type: 'string' }, //英文品名(畫面)
        { name: 'APPQTY', type: 'string' }, //申請數量(畫面)
        { name: 'ACT_PICK_QTY', type: 'string' }, //揀貨數量(畫面)
        { name: 'BASE_UNIT', type: 'string' } //撥補單位(畫面)
    ]
});

File: WebApp/Models/C/CD0005D.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class CD0005D : JCLib.Mvc.BaseModel
    {
        public string SEQ { get; set; } //項次(畫面)
        public string MMCODE { get; set; } //院內碼(畫面)
        public string MMNAME_C { get; set; } //中文品名(畫面)
        public string MMNAME_E { get; set; } //英文品名(畫面)
        public string APPQTY { get; set; } //申請數量(畫面)
        public string ACT_PICK_QTY { get; set; } //揀貨數量(畫面)
        public string BASE_UNIT { get; set; } //撥補單位(畫面)
    }
}
